micronaut-camunda-bpm-feature/src/main/java/info/novatec/micronaut/camunda/bpm/feature/CamundaBpmVersion.java

```diff
@@ -0,0 +1,21 @@
+package info.novatec.micronaut.camunda.bpm.feature;
+
+import org.camunda.bpm.engine.ProcessEngine;
+
+import java.util.Optional;
+
+public class CamundaBpmVersion {
```

Comment:
* Please add a comment like "// Implementation based on https://github.com/camunda/camunda-bpm-platform/blob/master/spring-boot-starter/starter/src/main/java/org/camunda/bpm/spring/boot/starter/util/CamundaBpmVersion.java"
* If you like you can add yourself as the author

See class ApplicationContextElResolver for an example for both